build_sys_common.rs

```diff
@@ -0,0 +1,31 @@
+use std::env::var;
+
+// Attempts to discover which library file to use and where it is located.
+//
+// If the `cfg(target_env)` is set to "MSVC" then it will use the MSVC
+// libraries, otherwise it will assume "GNU" and use the GCC libraries.
+//
+// If the environment variable `CSFML_HOME` is set, then it will look there
+// for the libraries, otherwise it defaults to `.`.
+//
+// It assumes that CSFML is simply unzipped - as is - into a folder, and the
+// libraries exist in: `$CSFML_HOME/lib/[msvc|gcc]/*.lib`.
+//
+pub fn lib_path(file: &str) -> String {
+    let lib = if cfg!(target_env = "msvc") {
+        format!("csfml-{}", file)
+    } else {
+        format!("libcsfml-{}", file)
```

Comment:
Downloading the OS X version also has them named `libcsfml-*.dylib`. I guess two questions:

1. Does rust auto-add the "lib" prefix? If so, then the MSVC check can go away and just leave the CSFML_HOME there (assuming you want it).
2. Is there something you'd like me to do to the code w/o being able to test it?